src/main/java/hu/oe/nik/szfmv/automatedcar/Main.java

```diff
@@ -33,11 +36,18 @@ private void init() {
         world = new World(5000, 3000);
 
         // create an automated car
+        // car = new AutomatedCar(3522, 2316, "car_2_white.png");
+        // car = new AutomatedCar(474, 669, "car_2_white.png");
```

Comment:
Ha ezekre nincs már szükség ki is lehet venni a kódból.

Reply:
Demózás miatt hagytam benne, ha gyorsan meg akarjuk nézni a különböző debug poligonokat.

Reply:
Oké, értem.